src/tools/dbcparser/Validator.h

```diff
@@ -21,6 +21,7 @@
 #include <regex>
 #include <utility>
 #include <cstddef>
+#include <ranges>
```

Comment:
<algorithm> is the correct header for this (https://en.cppreference.com/w/cpp/algorithm/ranges/find).

Reply:
Sure, not a problem. and the other one?

Reply:
Only here.